build.sbt

```diff
@@ -265,7 +255,7 @@ lazy val scalahostSbt = project
     buildInfoSettings,
     sbt.ScriptedPlugin.scriptedSettings,
     sbtPlugin := true,
-    bintrayRepository := "maven", // sbtPlugin overrides this to sbt-plugins
+    bintrayRepository := "sbt", // sbtPlugin overrides this to sbt-plugins
```

Comment:
I don't think this is necessary, sbt resolves plugins from maven central. I publish sbt-scalafmt to maven with no problems.